bindings/core/src/method_handler/wallet.rs

```diff
@@ -214,10 +217,14 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
                 let public_key = PublicKey::try_from_bytes(prefix_hex::decode(public_key_str)?)
                     .map_err(iota_sdk::wallet::Error::from)?;
                 wallet
-                    .prepare_implicit_account_transition(&output_id, Some(public_key))
+                    .prepare_implicit_account_transition(&output_id, public_key)
                     .await?
-            } else {
+            } else if let Some(bip_path) = bip_path {
```

Comment:
wait huh? this is stable now??

Reply:
I wasn't too convinced when I wrote it 😄